tests/test-definitions.txt

```diff
@@ -4,7 +4,7 @@
 
 
 # Single Server only Tests
-BackupAuthNoSysTests,BackupAuthSysTests,BackupNoAuthSysTests,arangosh,arangobench,permissions,paths_server,version,active_failover single name=single_server_only
+BackupAuthNoSysTests,BackupAuthSysTests,BackupNoAuthSysTests,arangosh,arangobench,permissions,paths_server,version,active_failover single name=single_server_only -- --extremeVerbosity true 
```

Comment:
Is this intentional or just some leftover?

Reply:
ah, debug. reverted.